matcher/service/openreview_interface.py

```diff
@@ -781,6 +785,10 @@ def __init__(
         self.probability_limits = float(
             self.config_note.content.get("randomized_probability_limits", 1.0)
         )
+        self.perturbation = float(
```

Comment:
same here

Reply:
Done for the UI variables. The variables names within the solver remain unchanged.